### DISCUSSION TITLE
Feature request: Bloom, Glare, Temperature, Light groups in Camera settings

### DISCUSSION THREAD

**Discussion by odil24** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#330] [Ideas]
Is there have any plans to add this kind of effects on Camera settings panel like LuxCoreRender did?

**Comment by bsavery** [comment]
Please add links or something to the luxcore documentation for what you're asking!

**Comment by odil24** [comment]
> 
> 
> Please add links or something to the luxcore documentation for what you're asking!

https://wiki.luxcorerender.org/LuxCore_SDL_Reference_Manual_v2.0#Image_pipeline_the_SDL_manual

**Comment by bsavery** [comment]
So we do support some of these things in the Radeon Image Filter Library, but wouldn't these be better done in the blender compositing nodes?
